Formula/theos.rb

```diff
@@ -0,0 +1,46 @@
+class Theos < Formula
+  desc "Cross-platform tools for developing software for iOS and other platforms"
+  homepage "https://theos.dev"
+  license "GPL-3.0-or-later"
+  head "https://github.com/theos/theos.git", branch: "master"
+
+  stable do
+    url "https://github.com/theos/theos.git",
+      branch:   "master",
+      revision: "e53ec51e92a3716def537c7ff1b5b2a86930700c"
+    version "2.6"
+  end
```

Comment:
You don't need a `stable` block for this. Also, we only use tagged releases. It looks like this is just an arbitrary commit. See https://docs.brew.sh/Acceptable-Formulae#stable-versions.